src/lib/full-path.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Get a PostCSS source full path
+ * @module lib/full-path
+ */
+
+const {join, dirname} = require('path');
+
+module.exports = (fp, rp) =>
```

Comment:
It's about readability. If you think that a one-liner is easy to read in the same way, make it one-line